depsdotdev.go

```diff
@@ -171,6 +178,70 @@ func (c *client) GetDirectDependencies(ctx context.Context, p Package) ([]Packag
 		})
 	}
 
+	if hasBundledDependencies {
+		if p.Ecosystem == "npm" {
+			bundledDependencies, err := c.getNPMBundledDependencies(ctx, versionKey)
+			if err != nil {
+				return nil, fmt.Errorf("getting bundled dependencies: %w", err)
+			}
+
+			result = append(result, bundledDependencies...)
+		}
+
+		// TODO log a warning if hasBundledDependencies is true and the ecosystem is NOT npm
```

Comment:
Remaining leftover?

Reply:
I still haven't implemented logging, and since this is a package and not an app it's a bit harder to do it right so I'll do it later.

I created #19 and added mentions to it in [c6144fb](https://github.com/DataDog/aggregated-dependency-score/pull/18/commits/c6144fb3d3f995ef787ace45cf6a65e69291cff5)